main/ejml-dsparse/test/org/ejml/sparse/csc/TestRandomMatrices_DSCC.java

```diff
@@ -62,6 +65,21 @@ void uniform() {
         assertTrue(CommonOps_DSCC.checkSortedFlag(a));
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = {"true", "false"})
+    void generateUniform(boolean sorted) {
+        DMatrixSparseCSC a = RandomMatrices_DSCC.generateUniform(numRows, numRows, 4, -1, 1, sorted, rand);
+
```

Comment:
Could you add a check on the number of entries? Doesn't need to be exact but you know the upper and low bound. Also run through a series of different matrix shapes, but keep the number of total elements relative small so it still runs fast.